docs/how-to-guides/adding-support-for-a-new-online-store.md

```diff
@@ -263,23 +305,104 @@ online_store: feast_custom_online_store.mysql.MySQLOnlineStore
 
 ## 4. Testing the OnlineStore class
 
-Even if you have created the `OnlineStore` class in a separate repo, you can still test your implementation against the Feast test suite, as long as you have Feast as a submodule in your repo. In the Feast submodule, we can run all the unit tests with:
+### Integrating with the integration test suite and unit test suite.
 
-```
-make test
-```
+Even if you have created the `OnlineStore` class in a separate repo, you can still test your implementation against the Feast test suite, as long as you have Feast as a submodule in your repo.
 
-The universal tests, which are integration tests specifically intended to test offline and online stores, can be run with:
+1. In the Feast submodule, we can run all the unit tests and make sure they pass:
+    ```
+    make test-python
+    ```
+
+
+2. The universal tests, which are integration tests specifically intended to test offline and online stores, should be run against Feast to ensure that the Feast APIs works with your online store.
+    - To run the integration tests, you must parametrize the integration test suite based on the `FULL_REPO_CONFIGS` variable defined in `sdk/python/tests/integration/feature_repos/repo_configuration.py` to use your own custom online store.
```

Comment:
This is kind of confusing to me. I don't think users should try to "run the integration tests" since that is for official plugins.

It's useful context to understand the second bullet though. Would rephrase that Feast parametrizes integration tests using the FULL_REPO_CONFIGS variable, which you overwrite to enable testing for the contrib offline store

Reply:
fixed